### PR TITLE
Handle exception while cleaning temporary media folder

### LINKED ISSUE #4662: AzureMediaStorage - The specified container does not exist
When you enable AzureMediaStorage feature on an empty Azure storage account, you get an exception : 'StorageException: The specified container does not exist.'

### Reproduction steps :
- Create a fresh new AzureStorageAccount
- Configure the storage account in `appsettings.json`
- Enable 'Azure Media Storage' feature

### PR DESCRIPTION
**Pull request by TFleury**
Fixes #4662

### CODE REVIEW

**Comment by deanmarcussen** [comment]
LGTM

File: src/OrchardCore.Modules/OrchardCore.Media/Services/TempDirCleanerService.cs [hunk 1]
```diff
@@ -28,26 +28,33 @@ public async Task ActivatedAsync()
...
+                    foreach (var c in contents)
                     {
-                        _logger.LogWarning("Temporary entry {0} could not be deleted.", c.Path);
+                        var result = c.IsDirectory ?
+                            await _fileStore.TryDeleteDirectoryAsync(c.Path)
+                            : await _fileStore.TryDeleteFileAsync(c.Path);
+
+                        if (!result)
+                        {
+                            _logger.LogWarning("Temporary entry {0} could not be deleted.", c.Path);
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Media/Services/TempDirCleanerService.cs]: Wrong usage of parameters in log message. I know it's not from your change

Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Media/Services/TempDirCleanerService.cs]:
```suggestion
                            _logger.LogWarning("Temporary entry {0} could not be deleted.", c.Path);
                            _logger.LogWarning("Temporary entry {Path} could not be deleted.", c.Path);
```

Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Media/Services/TempDirCleanerService.cs]:
```suggestion
                            _logger.LogWarning("Temporary entry {Path} could not be deleted.", c.Path);
```

**Pull request merged**